### PR TITLE
feat: add method to delete all history

### PR DESCRIPTION
**Pull request by anton7c3**
Delete all history from the browser using about:preferences and some javascript magic.

### CODE REVIEW

File: spec/marionette_spec.cr [hunk 1]
```diff
@@ -1,25 +1,105 @@
...
+      marionette.close_session
+    end
+
+    it "Clears browser history" do
+      webpage = <<-HTML
+      <html>
+        <head>
+          <script defer>
+            function probe_local_storage() {
+            var d = document.getElementById("message");
```
Comment (alonchin) [inline spec/marionette_spec.cr]:
```suggestion
              const d = document.getElementById("message");
```

File: spec/marionette_spec.cr [hunk 2]
```diff
@@ -1,25 +1,105 @@
...
+            var token = localStorage.getItem("token");
```
Comment (alonchin) [inline spec/marionette_spec.cr]:
```suggestion
              const token = localStorage.getItem("token");
```

**Review by alonchin — APPROVED** [review]

File: spec/marionette_spec.cr (hunk 1 above)
Comment (anton7c3) [reply spec/marionette_spec.cr]: fixed

File: spec/marionette_spec.cr (hunk 2 above)
Comment (anton7c3) [reply spec/marionette_spec.cr]: fixed

**Review by alonchin — APPROVED** [review]

**Pull request merged**